### PR TITLE
⭐️ updated aws policy

### PR DESCRIPTION
**Pull request by chris-rock**
This PR updates AWS policy:

- makes use of variants to keep the same checks for account and its sub-resources
- move asset filter to the query
- reformat the markdown text

### CODE REVIEW

File: core/mondoo-aws-security.mql.yaml [hunk 1]
```diff
@@ -10,132 +10,124 @@ policies:
...
+        
+        For a complete list of AWS region codes, see [Regions and Zones](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html).
+        
+        ### Scan an AWS account using a specific profile
+        
+        If multiple AWS profiles are configured for the AWS CLI, `cnspec` can target a specific profile by setting the `AWS_PROFILE` environment variable or the `--profile` command line flag.
+        
+        ```bash
+        export AWS_PROFILE=my-profile
+        nspec scan aws
```
Comment (atomic111) [inline core/mondoo-aws-security.mql.yaml:58]:
```suggestion
        cnspec scan aws
```

File: core/mondoo-aws-security.mql.yaml [hunk 2]
```diff
@@ -10,132 +10,124 @@ policies:
...
+        ### Prerequisites
+        
+        Remote scanning of AWS accounts with `cnspec` relies on the access key ID and secret access key configured for the AWS CLI. For more information on configuring these keys in the AWS CLI, see [Configuring the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html).
+        
+        ### Scan an AWS account
+        
+        The following command will perform a scan of all enabled regions in an AWS account:
+        
+        ```bash
+
```
Comment (tas50) [inline core/mondoo-aws-security.mql.yaml:37]:
```suggestion
```

File: core/mondoo-aws-security.mql.yaml [hunk 3]
```diff
@@ -10,132 +10,124 @@ policies:
...
+        Remote scanning of AWS accounts with `cnspec` relies on the access key ID and secret access key configured for the AWS CLI. For more information on configuring these keys in the AWS CLI, see [Configuring the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html).
+        
+        ### Scan an AWS account
+        
+        The following command will perform a scan of all enabled regions in an AWS account:
+        
+        ```bash
+        
+        cnspec scan aws
+
```
Comment (tas50) [inline core/mondoo-aws-security.mql.yaml:39]:
```suggestion
```

File: core/mondoo-aws-security.mql.yaml (hunk 1 above)
Comment (tas50) [inline core/mondoo-aws-security.mql.yaml:58]:
```suggestion
        cnspec scan aws
```

File: core/mondoo-aws-security.mql.yaml [hunk 4]
```diff
@@ -425,80 +867,236 @@ queries:
...
+          }
+        ]
+        ```
       remediation: |
         __Terraform__
 
-        The following Terraform resource configures account level access to S3:
+        Terraform provides the resource `aws_default_security_group`, which differently from other Terraform resources, has the following effects in the state of the infrastructure.
+
+        1. \"Adopts\" the default security group for the provided `vpc_id`.
```
Comment (tas50) [inline core/mondoo-aws-security.mql.yaml:983]: Should these all have the \s in them?

**Review by atomic111 — APPROVED** [review]

**Pull request merged**